server/routes/publicAccountCreation.js

```diff
@@ -0,0 +1,19 @@
+const router = express.Router();
+
+ const accountCreationHandler = require('../models/handlers/accountCreation');
+
+ router.get('/api/create-user', async(req, res) => {
```

Comment:
I would move these lines to the public route to improve the codebase as it provides a "public" action similar to login, i.e., it would allow anyone to access this endpoint whereas an endpoint such as `book-appointment` should only be accessed by verified users of the system.

Reply:
Hi Amy,
Thank you for suggesting that!
**My action:** I've added this to the public.js file.
You can now see it in line 30-48 of server/test/accountCreation.test.js

Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy, thank you for moving the code. I would also remove this file as it is no longer needed.

Reply:
Hi Amy, once you confirm the comment on public.js file, I can delete this!

Reply:
Hi Nancy, feel free to remove this file as the main contents have already been copied to the public.js file.